html/browsers/browsing-the-web/navigating-across-documents/initial-empty-document/iframe-src-204-fragment.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>Fragment navigations on iframe with src set to URL that doesn't load a new document</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/helpers.js"></script>
+<body></body>
+<script>
+"use strict";
+const url1 = "about:blank#1";
+const url2 = "/common/blank.html?2";
+
+promise_test(async t => {
+  const startingHistoryLength = history.length;
+  // Create an iframe with src set to URL that doesn't load a new document, so
+  // it will stay in the initial empty document.
+  const iframe = insertIframeWith204Src(t);
+  assert_equals(history.length, startingHistoryLength,
+    "Inserting iframe with src set to URL that doesn't load a new document must not change history.length");
+
+  // Do a fragment navigation within the initial empty document through iframe.src.
+  iframe.src = url1;
+  await new Promise(resolve => setTimeout(setTimeout(resolve, 100), 100));
```

Comment:
`setTimeout()` takes a function, so passing it the result of `setTimeout()` (an integer) doesn't really work. I think this should be just `await new Promise(resolve => setTimeout(resolve, 100))`

Reply:
Ah, of course. Thank you!